templates/block.html

```diff
@@ -28,3 +28,11 @@ <h2>{{"Transaction".tally(self.block.txdata.len())}}</h2>
   <li><a href=/tx/{{txid}}>{{txid}}</a></li>
 %% }
 </ul>
+%% if self.inscriptions.len() > 0 {
```

Comment:
I think inscriptions should be above transactions. Maybe we should also preview them like on the inscriptions page